README.md

```diff
@@ -1,77 +1,67 @@
 
-### Spares
 
-The player has a spare if the knocks down all 10 pins with the two rolls of a frame. The bonus for that frame is the number of pins knocked down by the next roll (first roll of next frame).
+  - **Bonus scoring:**
 
-### 10th frame
+    - As a user, if I get a strike or a spare I wont know my proper total for that round until the next round is over
+    - As a user, if I get a strike my next two rolls are added as a bonus to the score for the strike round
+    - As a user, if i get a spare, my next roll is added as a bonus to the score for the spare round
+    - As a user, if I roll 12 strikes in a row I should score 300 points
 
-If the player rolls a strike or spare in the 10th frame they can roll the additional balls for the bonus. But they can never roll more than 3 balls in the 10th frame. The additional rolls only count for the bonus not for the regular frame count.
 
-    10, 10, 10 in the 10th frame gives 30 points (10 points for the regular first strike and 20 points for the bonus).
-    1, 9, 10 in the 10th frame gives 20 points (10 points for the regular spare and 10 points for the bonus).
+  - **10th Frame**:
 
-### Gutter Game
+    - As a user if I have not rolled a special roll on the 10th frame, the game is over
+    - As a user I want to see my final total when the game has ended
+    - As a user if I get any bonus rolls they still count as being within the 10th frame, there cannot be more than 10 frames
+    - As a user if I roll a strike in the 10th frame, I get to do two extra rolls to work out my bonuses
+    - As a user if I roll a spare in the 10th frame, I get to do 1 extra roll to work out my bonuses
+    - As a user if I get a strike in the 10th frame, I get two rolls even if one of them is a strike
 
-A Gutter Game is when the player never hits a pin (20 zero scores).
+## Information that needs to be recorded:
+  - Number of current frame
+  - Number of rolls taken
+  - How many pins knocked down on all rolls
+  - Running total of the score
+  - Whether a spare or strike has taken place
 
-### Perfect Game
-
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
-
-In the image below you can find some score examples.
-
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
-
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
-
-## Code Review
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Note that referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want.
+## Proposed Classes:
```

Comment:
Might be good to update to show what was implemented and what is still proposed.